todo/src/reducers/todos.js

```diff
@@ -0,0 +1,53 @@
+import {
+  ADD_TODO,
+  TOGGLE_TODO,
+  UPDATE_TODO,
+  DELETE_TODO,
+  CLEAR_TODOS
+} from '../actions';
+
+const initialState = [
+  {
+    task: 'Bake Cookies',
+    id: 1528817084358,
+    completed: false
+  }
+];
+
+function todo(state, action) {
+  switch (action.type) {
+    case ADD_TODO:
+      return {
+        task: action.task,
+        id: Date.now(),
```

Comment:
just about the only thing on this app I'd actually change -- by nature functions like Date.now(), Math.random(), or library helpers with randomized results like uuid are impure since they will return different results upon multiple invocations. In the interest of maintaining pure functionality throughout, the new task created here should be handled within the action creator instead